js/utils.js

```diff
@@ -1311,7 +1475,10 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
       return {
         column: option.column,
         condition: option.logic,
-        value: option.value
+        filterModifier: option.filterModifier,
```

Comment:
I don't understand the purpose of `filterModifier`, how it's used and why it's necessary.

Reply:
This variable has many necessary values such as offset, oseTimezone and others. They are required for every filter.